src/content/docs/api/osc-and-ws.mdx

```diff
@@ -13,43 +13,120 @@ Get in touch on email [mail@getontime.no](mailto:mail@getontime.no) or [open an
+| /load/id/[id]          |                                                            | "success"                       | Loads event with given ID                                            |
+| /load/cue/[cue]        |                                                            | "success"                       | Loads next event with given cue                                      |
+| /pause                 |                                                            | "success"                       | Pauses playback                                                      |
+| /stop                  |                                                            | "success"                       | Stops playback and unloads the loaded event                          |
+| /reload                |                                                            | "success"                       | Reloads selected event                                               |
+| /roll                  |                                                            | "success"                       | Sets playback mode to roll                                           |
+| /addtime               | ?add=[number] or ?remove=[number]                          | "success"                       | Adds time to a playing event (in seconds)                            |
+| /extratimer/1/start    |                                                            | [TimerState](../)               | Starts extratimer1                                                   |
+| /extratimer/1/stop     |                                                            | [TimerState](../)               | Stops extratimer1                                                    |
+| /extratimer/1/pause    |                                                            | [TimerState](../)               | Pauses extratimer1                                                   |
+| /extratimer/1          | ?duration=[number]&direction=["count-up" \| "count-down" ] | [TimerState](../)               | Sets duration and/or direction of extratimer1                        |
+
+</TabItem>
+
+<TabItem label="WS">
+:::note[Notes on WS]
+:::
+
+| ???                    | Return                          | Description                                                          |
+| ---------------------- | ------------------------------- | -------------------------------------------------------------------- |
+| \{ type: 'version' \}  | \{ payload: '3.0.0-alpha' \}    |                                                                      |
+| /poll                  | [RuntimeStore](../runtime-data) |                                                                      |
+| /change/[id]           | "changes pending"               | changes the [property](../api-details) of event with `ID` to `value` |
```

Comment:
should we omit "changes pending" since it offers no value? I cant infer from here a success or failure, just meant the message was received and maybe something will happen